pysal/frozen.py

```diff
@@ -2,19 +2,11 @@
 Frozen subpackages for meta release.
 """
 
-frozen_packages = {    "libpysal": "4.2.2",
-    "esda": "2.2.1",
-    "giddy": "2.3.0",
-    "inequality": "1.0.0",
-    "pointpats": "2.1.0",
-    "segregation": "1.2.0",
-    "spaghetti": "1.4.1",
-    "mgwr": "2.1.1",
-    "spglm": "1.0.7",
-    "spint": "1.0.6",
-    "spreg": "1.0.4",
-    "spvcm": "0.3.0",
-    "tobler": "0.2.0",
-    "mapclassify": "2.2.0",
-    "splot": "1.1.2"
-}
\ No newline at end of file
+frozen_packages = {"libpysal": "4.2.2", "esda": "2.2.1",
```

Comment:
Will these (and those in `environment.yml` be updated to currently released packages? On first glance the versions of `libpysal` and `spaghetti` are not the latest.

Reply:
Yes.